toolchain/sem_ir/formatter.cpp

```diff
@@ -417,6 +417,10 @@ auto Formatter::FormatInterface(InterfaceId id, const Interface& interface_info)
     FormatArg(interface_info.associated_entities_id);
     out_ << "\n";
 
+    IndentLabel();
+    out_ << "!requires:\n";
```

Comment:
It seems like this could be part of `FormatRequireImplsBlock` too; I get that it's outside on `!members`, but also `FormatNameScope` is used for other blocks.